src/components/Button/Button.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+import { DEFAULT_PROPS } from './Button.constants';
+import { StyledButton, LoadingSpinner, ButtonContent } from './Button.styles';
+
+import type { ButtonProps } from './Button.types';
+
+/** Button con variantes, tamaños y formas (MUI base + estilos custom) */
```

Comment:
Component JSDoc comment is in Spanish. Project enforces English across code and comments.

**Sugerencia:**
```typescript
Translate the comment to English, e.g. `/** Button with variants, sizes and shapes (MUI base + custom styles) */`
```